Update contributors.md (#429)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/contributors.md

> ## Steering Committee Members

Reviewer: Suggested order:
## Working Group
### Maintainers
...

### Members
...

## Steering Committee
...

Thanks to the following FOCUS Steering Committee Members for their contributions to this specification.

* Anne Johnston (Capital One)
* Michael Flanakin (Microsoft)
* Mike Fuller (FinOps Foundation)
* Roy Wolman (Amazon Web Services)
* Sarah McMullin (Google)
* Tim O'Brien (Walmart)

> * Udam Dewaraja (FinOps Foundation)

Reviewer: ```suggestion
```

## Maintainers

Thanks to the following FOCUS Maintainers for their contributions to this specification.

> * Anne Johnston (Capital One)
> * Christopher Harris (Datadog)
> * Dennis Park-Rodriguez (Amazon Web Services)
> * Erik Peterzon (CloudZero)
> * Irena Jurica (Neos)
> * Joaquin Prado (FinOps Foundation)
> * Karl Kraft (Walmart)
> * Larry Advey (Twilio)
> * Michael Flanakin (Microsoft)
> * Mike Fuller (FinOps Foundation)
> * Riley Jenkins (DOMO)
> * Roy Wolman (Amazon Web Services)
> * Rupa Patel (Google)
> * Sarah McMullin (Google)
> * Tim O'Brien (Walmart)
> * Udam Dewaraja (FinOps Foundation)

Reviewer: ```suggestion
* Christopher Harris (Datadog)
* Dennis Park-Rodriguez (Amazon Web Services)
* Erik Peterzon (CloudZero)
* Irena Jurica (Neos)
* Joaquin Prado (FinOps Foundation)
* Karl Kraft (Walmart)
* Larry Advey (Twilio)
* Michael Flanakin (Microsoft)
* Mike Fuller (FinOps Foundation)
* Riley Jenkins (DOMO)
* Rupa Patel (Google)
* Udam Dewaraja (FinOps Foundation)
```

> 

Reviewer: Seems like we would want to at least not double count members between maintainers and members

Author: Was thinking the same but don't know a good way to manifest this other than 1 list like:

```
John Doe
Jane Doe [Maintainer]
Mike Doe [Steering Committee]
Alex Doe [Maintainer, Steering Committee]
```

Author: Will just remove Maintainers from Member section for now.

## Working Group

### Maintainers

Thanks to the following FOCUS Maintainers for their contributions to this specification.

* Anne Johnston (Capital One)
* Christopher Harris (Datadog)
* Dennis Park-Rodriguez (Amazon Web Services)

> * Erik Peterzon (CloudZero)

Reviewer 2: Correcting Erik's name
```suggestion
* Erik Peterson (CloudZero)
```

* Irena Jurica (Neos)
* Joaquin Prado (FinOps Foundation)
* Karl Kraft (Walmart)
* Larry Advey (Twilio)
* Michael Flanakin (Microsoft)
* Mike Fuller (FinOps Foundation)

> * Riley Jenkins (DOMO)

Reviewer 3: Marketing reminded me it isn't DOMO...

## Working Group

### Maintainers

> Thanks to the following FOCUS Maintainers for their leadership of the FOCUS specification.

Reviewer: ```suggestion
Thanks to the following FOCUS Maintainers for their leadership and contributions to the FOCUS specification.
```

* Christopher Harris (Datadog)
* Dennis Park-Rodriguez (Amazon Web Services)
* Erik Peterson (CloudZero)
* Irena Jurica (Neos)
* Joaquin Prado (FinOps Foundation)
* Karl Kraft (Walmart)
* Larry Advey (Twilio)
* Michael Flanakin (Microsoft)
* Mike Fuller (FinOps Foundation)
* Riley Jenkins (DOMO)
* Rupa Patel (Google)

> * Tim O'Brien (Walmart)

Reviewer 4: Sorry, Tim! 😶‍🌫️
```suggestion
```

* Udam Dewaraja (FinOps Foundation)

### Contributors

Thanks to the following FOCUS members for their contributions to the FOCUS specification.

* Adam Schwartz (Ernst & Young)
* Alex Hullah (Goldman Sachs)
* Amit Wadhwa (Google)
* Anand Tripathi (Amazon Web Services)
* Anderson Oliveira (Farfetch.com)
* Andrew Qu (Everest)

> * Arun Ramakrishnan (Oracle)

Reviewer 4: Bring Anne back
```suggestion
* Anne Johnston (Capital One)
* Arun Ramakrishnan (Oracle)
```

* Ben Shy (Microsoft)
* Ben Swoboda (NetApp)
* Ben Zhou (Apptio)
* Casey Doran (Apptio)
* Chandra Devarajan (CloudTrakr)
* Chew Esmero (Alphaus Inc.)
* Colin Jack (Snow Software)
* Deeja Cruz (Datadog)
* Eleni Rundle (Google)
* Graham Murphy (Australian Retirement Trust)
* Harish Kumar Munagapat (Kyndryl)
* Hrishikesh Sardar (Kyndryl)
* Ilia Semenov (CloudThread)
* Jacky Liu (Google Cloud)
* Jacqui Wilson (Old Mutual South Africa)
* Janine Pickard-Green (MagicOrange Group Limited)
* Jason Kelly (Anglepoint Group Inc)
* Joe Ferrero (DB Gurus Inc)
* John Grubb (Platform.sh)
* Joshua Kwan (Ternary)
* Letian Wang (Atlassian)
* Luna Bernal (Twilio)
* Marc Perreaut (Amadeus)
* Marcin Walkow (Nordcloud)
* Mark Krawczeniuk (NetApp)
* Matt Ray (Kubecost)
* Michael Arkoosh (Vega)
* Mike Polson (VMWare)
* Nick Kotze (Surveil)
* Nicolas Fonrose (Teevity)
* Peter Marton (OpenMeter.io)
* Ray Ding (Accenture)
* Ricardo Triana (Accenture)
* Rodney Joyce (CloudMonitor)
* Sanjna Srivatsa (VMWare)
* Shawn Alpay (Envisor)
* Sireesha Oram (Kyndryl)
* Sonal Garg (Kyndryl)
* Sumaira Nazir (Platform.sh)
* Sureshkrishna Gonugunta (Box)
* Tatiana Dubovchenko (Flexera)

> * Trey Morgan (Microsoft)

Reviewer 4: ```suggestion
* Tim O'Brien (Walmart)
* Trey Morgan (Microsoft)
```

## Working Group

### Maintainers

Thanks to the following FOCUS Maintainers for their leadership and contributions to the FOCUS specification.

* Christopher Harris (Datadog)
* Dennis Park-Rodriguez (Amazon Web Services)
* Erik Peterson (CloudZero)
* Irena Jurica (Neos)
* Joaquin Prado (FinOps Foundation)
* Karl Kraft (Walmart)
* Larry Advey (Twilio)
* Michael Flanakin (Microsoft)
* Mike Fuller (FinOps Foundation)

> * Riley Jenkins (DOMO)

Reviewer: ```suggestion
* Riley Jenkins (Domo)
```

* Casey Doran (Apptio)
* Chandra Devarajan (CloudTrakr)
* Chew Esmero (Alphaus Inc.)
* Colin Jack (Snow Software)
* Deeja Cruz (Datadog)
* Eleni Rundle (Google)

> * Graham Murphy (Australian Retirement Trust)

Reviewer: ```suggestion
* George Parker (Salesforce)
* Graham Murphy (Australian Retirement Trust)
```

* Harish Kumar Munagapat (Kyndryl)
* Hrishikesh Sardar (Kyndryl)
* Ilia Semenov (CloudThread)
* Jacky Liu (Google Cloud)
* Jacqui Wilson (Old Mutual South Africa)
* Janine Pickard-Green (MagicOrange Group Limited)
* Jason Kelly (Anglepoint Group Inc)
* Joe Ferrero (DB Gurus Inc)
* John Grubb (Platform.sh)
* Joshua Kwan (Ternary)
* Letian Wang (Atlassian)
* Luna Bernal (Twilio)
* Marc Perreaut (Amadeus)
* Marcin Walkow (Nordcloud)
* Mark Krawczeniuk (NetApp)
* Matt Ray (Kubecost)
* Michael Arkoosh (Vega)
* Mike Polson (VMWare)
* Nick Kotze (Surveil)
* Nicolas Fonrose (Teevity)
* Peter Marton (OpenMeter.io)
* Ray Ding (Accenture)
* Ricardo Triana (Accenture)
* Rodney Joyce (CloudMonitor)
* Sanjna Srivatsa (VMWare)
* Shawn Alpay (Envisor)
* Sireesha Oram (Kyndryl)
* Sonal Garg (Kyndryl)
* Sumaira Nazir (Platform.sh)
* Tatiana Dubovchenko (Flexera)
* Tim O'Brien (Walmart)
* Trey Morgan (Microsoft)
* Trig Ghosh (Accenture)
* Venu Potanker (Ernst & Young)
* Webb Brown (Kubecost)
* Yuriy Prykhodko (Amazon Web Services)
* Zach Erdman (Amazon Web Services)

## Steering Committee Members

> Thanks to the following FOCUS Steering Committee Members for their contributions to the FOCUS specification.

Reviewer: ```suggestion
Thanks to the following FOCUS Steering Committee members for their leadership on the FOCUS specification.
```
